src/lib/decorators.test.ts

```diff
@@ -59,7 +59,7 @@ describe('decorators', () => {
     const seed1 = '00000000000000000000000000000My1';
     const verkey = await indy.createKey(wh, { seed: seed1 });
 
-    const result = await sign(wh, message, 'connection', verkey);
+    const result = await sign(wh, message, 'connection', verkey, true);
```

Comment:
We should try to avoid flag parameters (`true/false`) see https://martinfowler.com/bliki/FlagArgument.html. In this case, it's also related only to the testing environment and it could confuse regular function consumers. I would suggest two options:

1. Factory function `sign(timestamp)(wh, message, 'connection', verkey)`
2. Extract it into separate module (let's call it `timestamp` until we find better name) and than mock that with `jest.mock('./timestamp');` in tests

I think 2. option is simpler right now and it keeps common API with its counterpart `verify`

Reply:
I briefly looked at [Jest docs](https://jestjs.io/docs/en/jest-object#jestmockmodulename-factory-options) and something like this should work:
```
jest.mock('./timestamp', () => {
  return jest.fn(() => Uint8Array.of(0, 0, 0, 0, 0, 0, 0, 0));
});
```